views/overview/tabs.jade

```diff
@@ -0,0 +1,17 @@
+include ../mixins/overviewFields
+
+if (tabs && tabs.length)
+    div.tabs
+        div
+            ul.nav.nav-tabs.nav-tabs-overview
+                each tab, index in tabs
+                    li(class="#{index === 0 ? 'active' : ''}")
+                        a(href='#' + tab.label + index, role='tab' data-toggle='tab')
+                            span=tab.label
+        div.tab-content
+            each tab, index in tabs
+                div(id=tab.label + index, class="#{index === 0 ? 'tab-pane active' : 'tab-pane'}")
+                    if (tab.fields)
```

Comment:
This should check `if (tab.body)` first so it's consistent with your code logic in overview.render.